src/core/selinux-access.c

```diff
@@ -179,6 +181,7 @@ int mac_selinux_generic_access_check(
                 sd_bus_message *message,
                 const char *path,
                 const char *permission,
+                const char *func,
```

Comment:
Can we please also call this `function` here? It's jarring that it's different above and here.